docs/intro/installation.rst

```diff
@@ -18,7 +18,7 @@ In general, the DataLad installation requires Python 3 (see the
 why this is required), :term:`Git`, and :term:`git-annex`, and for some
 functionality `7-Zip <https://7-zip.org>`_.  The instructions below detail how
 to install the core DataLad tool and its dependencies on common operating
-systems. They do not cover the various :term:`DataLad extension`'s that need to be installed separately, if desired.
+systems. They do not cover the various :term:`DataLad extension`s that need to be installed separately, if desired.
```

Comment:
I think this breaks the build, I'll toy around with it locally

Reply:
```suggestion
systems. They do not cover the various :term:`DataLad extension`\s that need to be installed separately, if desired.
```

Reply:
Sorry, I didn't check the build locally. I think other places manage to work with an s but no apostrophe? In either case, the apostrophe made it to the PDF version I saw.